Cargo.toml

```diff
@@ -35,7 +35,7 @@ anyhow = { version = "1.0", optional = true }
 chrono = { version = "0.4", optional = true }
 eyre = { version = ">= 0.4, < 0.7", optional = true }
 hashbrown = { version = ">= 0.9, < 0.13", optional = true }
-indexmap = { version = ">= 1.6, < 1.8", optional = true }
+indexmap = { version = ">= 1.6, <= 1.9", optional = true }
```

Comment:
Given how semver is supposed to work beyond 1.x, I think this should just become `1.6`, which would be equivalent to `^1.6` and hence `>= 1.6, < 2`. (It is different for the other optional dependencies which stall have 0.x versions.)

Reply:
Good point :), fixed